azure-py-aks/__main__.py

```diff
@@ -0,0 +1,125 @@
+import pulumi
+from pulumi import ResourceOptions
+from pulumi_azure.core import ResourceGroup
+from pulumi_azure.role import Assignment
+from pulumi_azure.ad import Application, ServicePrincipal, ServicePrincipalPassword
+from pulumi_azure.containerservice import KubernetesCluster, Registry
+from pulumi_azure.network import VirtualNetwork, Subnet
+
+config = pulumi.Config('azure-py-kubernetes')
+PREFIX = config.get('prefix') or 'replaceme'
+PASSWORD = config.get('password') or 'DEf34tg!rg35y#F23'
+SSHKEY = config.get('sshkey') or 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCxinIAIDDCradZPAgX5GzBLv00u4rigOLUbU00E44FrfMTqu5wXiejJ4ycSb1bI+//ZNgaB2UYRbPL7A9OUKY+K4sX5O84Q6DPMjo/90IANHVTLf3xTaSc7hpvXOtIjJTJeiamxClgnTAcR55RV/j9/Wptxa8GGcRmRCcSmJUkx5AZTFI+s8aF0W3aeHHRw7TxNKBuwrX7FDcHyGKvdkFg4OP863Xe5hp5ql1C3XibmCOp1CMPIU2hCmGOy1LGbOf/Pa+QKAdtUSrPNK/jBWvPWo0k02Ii0JtMAdlpVqnJc3czNIp5gEqZCRCGEdkb/kZnJiMRZhmLBYnC8tiMxvZj core@k8s'
```

Comment:
You probably don't want your public SSH key in this PR. :) For all of these config valiables, I think we should just use `config.required(...)` instead.

Reply:
its a random one I generated for folks so they dont have to do it. the idea here (because its a learning experience) that it will just work. I can make them required

Reply:
Makes sense, if you do make it required I think you can probably tell people to do something like `pulumi config set sshkey < ~/.ssh/id_rsa.pub` and it should be about as easy.